lib/gui/server.ts

```diff
@@ -14,11 +14,6 @@ import {ServerReadyData} from './api';
 import {BrowserFeature, ToolName} from '../constants';
 import type {TestplaneToolAdapter} from '../adapters/tool/testplane';
 import {ToolRunnerTree} from './tool-runner';
-import {
-    getSnapshotHashWithoutAttempt,
-    snapshotsInProgress,
-    TestContext
-} from '../adapters/event-handling/testplane/snapshots';
```

Comment:
Testplane is connected inside this module, but it doesn't exist in project with playwright for example. So I use `await import` below in the code only for testplane tool